helm-chart/kuberay-apiserver/templates/deployment.yaml

```
@@ -43,6 +43,35 @@ spec:
           httpGet:
             path: /healthz
             port: http
+      {{- if .Values.security }}
```

Can we have something like this to enable security only if `--set security.enable=true` flag is used with helm install
```suggestion
      {{- if .Values.security.enable }}
```

---

Does this mean the helm install from remote repo would always have the security sidecar?